examples/dummy_adaptive.py

```
@@ -4,11 +4,11 @@
 
 from impress import PipelineSetup
 from impress import ImpressBasePipeline
-from impress import ImpressManager
+from impress.impress_manager import ImpressManager
 
 from concurrent.futures import ThreadPoolExecutor
 from radical.asyncflow import ConcurrentExecutionBackend
-
+from agnostic_agent import LLMAgent
```

Ruff (0.12.2)

`agnostic_agent.LLMAgent` imported but unused

Remove unused import: `agnostic_agent.LLMAgent`

(F401)